src/main/connector/system/Linux/Linux.yaml

```diff
@@ -0,0 +1,156 @@
+          # memTotal;memFree;memUsed;memBuffers;memCached;memFreeUtilization;memUsedUtilization;memBuffersUtilization;memCachedUtilization
+          type: osCommand
+          commandLine: cat /proc/meminfo
+          computes:
+            - type: awk
+              script: "${file::memory.awk}"
+      mapping:
+        source: "${source::monitors.memory.simple.sources.source(1)}"
+        attributes:
+          id: memory_usage
+          system.memory.limit: $1
+        metrics:
+          system.memory.usage{system.memory.usage="free"}: $2
+          system.memory.usage{system.memory.usage="used"}: $3
+          system.memory.usage{system.memory.usage="buffers"}: $4
+          system.memory.usage{system.memory.usage="cached"}: $5
+          system.memory.utilization{system.memory.usage="free"}: $6
+          system.memory.utilization{system.memory.usage="used"}: $7
+          system.memory.utilization{system.memory.usage="buffers"}: $8
+          system.memory.utilization{system.memory.usage="cached"}: $9
+  disk:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # id;reads;readsMerged;readTime;writes;writesMerged;writeTime;ioTime
+          type: osCommand
+          commandLine: cat /proc/diskstats
+          computes:
+            - type: awk
+              script: '{print $3 ";" $4 ";" $5 ";" $7 / 1000 ";" $8 ";" $9 ";" $11 / 1000 ";" $13 / 1000}'
+      mapping:
+        source: "${source::monitors.disk.simple.sources.source(1)}"
+        attributes:
+          id: $1
+          system.disk.device: $1
+        metrics:
+          system.disk.operations{direction:read}: $2
+          system.disk.merged{direction:read}: $3
+          system.disk.operation_time{direction:read}: $4
+          system.disk.operations{direction:write}: $5
+          system.disk.merged{direction:write}: $6
+          system.disk.operation_time{direction:write}: $7
+          system.disk.io_time: $8
+  service:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # serviceName;load;active;sub;description
+          type: osCommand
+          commandLine: systemctl list-units --type=service --all
+          computes:
+            - type: awk
+              script: /\.service/ {sub(/^[^[:alnum:]]+\s*/, ""); sub("\.service", ""); printf $1 ";" $2 ";" $3 ";" $4 ";"; for (i=5; i<=NF; i++) printf "%s ", $i; printf "\n"}
+      mapping:
+        source: "${source::monitors.service.simple.sources.source(1)}"
+        attributes:
+          id: $1
+          description: $5
+        metrics:
+          system.service.loaded: $2
+          system.service.active: $3
+          system.service.sub: $4
+  filesystem:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # filesystem;mountpoint;type;used;available
+          type: osCommand
+          commandLine: /usr/bin/df -B1 --output=source,target,fstype,used,avail 
+          computes:
+            - type: awk

```

Comment:
This AWK could be replaced with a simple `keepOnlyMatchingLines` filter compute.

Then the mapping would need to be updated with:

```yaml
id: $1 ($2)
system.filesystem.device: $1 ($2)
system.filesystem.mountpoint: $3
system.filesystem.type: $4
```

and `metrics` mapping would use `$5` and `$6`.
```